### ISSUE TITLE
Windows Timeline entries not clickable/available (grayed out)

### ISSUE THREAD

**Issue by wazybr** [microsoft/vscode#98476]
- VSCode Version: 1.45.1 (user setup)
- OS Version: Windows_NT x64 10.0.18363

Steps to Reproduce:

1. Open any file with VSCode from Windows Explorer's "open with" menu.
2. Close VSCode.
3. Press Win+Tab to open Windows Timeline.
4. See that the Timeline entry for the file you just opened is grayed out and not clickable.

Does this issue occur when all extensions are disabled?: Yes

Screenshot:
![image](https://user-images.githubusercontent.com/49368302/82763467-6326a900-9dde-11ea-8260-b68238460f77.png)

**Comment by bpasero** [comment]
I have no knowledge about this feature or how it works.

@deepak1556 any clues?

I do not expect anything actionable from our side, but let's leave this issue open for a week to see if someone has ideas.

**Comment by wazybr** [comment]
Basically it's like a history of my activities on applications. I should be able to click one of those entries and VSCode should open the file or workspace I was working on.

I don't know much about the API but I think it's somehow tied to the jump list (the history of opened files that shows when you right-click VSCode's entry on start menu).

**Comment by bpasero** [comment]
We do add to the jump list and that seems to work just fine.

**Comment by wazybr** [comment]
I just realized the issue is happening with the jump list too. 

When I try to open my workspace file from the jump list, it says the file couldn't be located and shows an option to remove it from the jump list. But the file exists, as you can see in the screenshot below:
![image](https://user-images.githubusercontent.com/49368302/83341844-64644400-a2be-11ea-9064-ea47bd0fe494.png)

Should I rename the issue?

**Comment by bpasero** [comment]
Can you try to reproduce with our nightly insider builds? You can give our preview releases a try from: https://code.visualstudio.com/insiders/

**Comment by wazybr** [comment]
So, I already had the latest insider build installed and the behavior was the same, so I tried reinstalling it. After reinstalling a new shortcut was added to the Start menu and the old one was kept there because had moved it to the root of the Start menu (instead of inside a Visual Studio Code folder). All the jump list-related issues I was having were fixed, and even a new section called "Recent workspaces" popped in, but not in the old shortcut. The same happened after a reinstall of the stable build. See:

Old shortcut:
![image](https://user-images.githubusercontent.com/49368302/83355891-a16d1c80-a338-11ea-843d-203b90fcd115.png)

New shortcut:
![image](https://user-images.githubusercontent.com/49368302/83355897-a8942a80-a338-11ea-8ce5-3a690728d180.png)

Shouldn't the two shortcut have the same items in the jump list as they point to the same executable with the same arguments (no arguments in this case)? Even the working directory is the same:
![image](https://user-images.githubusercontent.com/49368302/83355961-29532680-a339-11ea-91d9-43f16096b912.png)

Well, after the reinstall, Windows Timeline behavior is fixed too, as long as I open VSCode using the new shortcut:
![image](https://user-images.githubusercontent.com/49368302/83355887-987c4b00-a338-11ea-97c5-e681da70bee4.png)

I'm now afraid of moving the shortcut to the root of the Start menu again.
